index.html

```diff
@@ -31,11 +31,11 @@ <h2>I'm a Software Developer</h2>
         </div>
 
         <ul>            
-            <li><img src="images/Vectorgithub.svg" alt="github-icon"></li>
-            <li><img src="images/Grouplinkedin.svg" alt="linkedin-icon"></li>
-            <li><img src="images/Vectorpeace.svg" alt="peace-icon"></li>
-            <li><img src="images/Vectortwitter.svg" alt="twitter-icon"></li>
-            <li><img src="images/Vectorem.svg" alt="M-icon"></li>
+            <a href="https://github.com/Chimoneg27"><li><img src="images/Vectorgithub.svg" alt="github-icon"></li></a>
+            <a href="https://www.linkedin.com/in/garvin-chimone-37208924a/"><li><img src="images/Grouplinkedin.svg" alt="linkedin-icon"></li></a>
+            <a href="https://www.angellist.com/"><li><img src="images/Vectorpeace.svg" alt="peace-icon"></li></a>
+            <a href="https://twitter.com/GarvinChimone"><li><img src="images/Vectortwitter.svg" alt="twitter-icon"></li></a>
+           <a href="https://medium.com/@chimonegarvin27"><li><img src="images/Vectorem.svg" alt="M-icon"></li></a>
         </ul>
```

Comment:
- [ ] Good job here for the adjustment but the correct way is to place the `a` tag inside `li` and not outside of it. so consider fixing it.
the structure would look something like this:
```
<ul class="hero-icons d-flex">
      <li>
            <a target="_blank" href="https://twitter.com/Zuhaib042" rel="noopener noreferrer"><img
              src="./images/twitter.svg" alt="twitter logo"></a>
      </li>
</ul>
```